src/components/OrganizationRenderOption.tsx

```diff
@@ -0,0 +1,27 @@
+import { Typography } from '@mui/material';
+import { HTMLAttributes } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Organization } from '../types/organization.types';
+import { getLanguageString } from '../utils/translation-helpers';
+
+interface OrganizationRenderOptionProps {
+  props: HTMLAttributes<HTMLLIElement>;
+  option: Organization;
+}
+
+export const OrganizationRenderOption = ({ props, option }: OrganizationRenderOptionProps) => {
+  const { i18n } = useTranslation();
+  const currentLanguage = i18n.language;
+
+  return (
+    <li {...props} key={option.id}>
+      <div>
+        <Typography fontWeight="bold">{getLanguageString(option.labels)}</Typography>
+        <Typography>
+          {option.country && `${option.country} | `}
+          {getLanguageString(option.labels, currentLanguage === 'nob' || currentLanguage === 'nno' ? 'en' : 'no')}
```

Comment:
opplevde en case i min PR at for institusjoner som bare har tekst for `nb`, så vil `getLanguageString` defaulte til den, så her vil vi i flere tilfeller få samme norske oversettelse flere ganger.
Jeg løste det sånn:
```
<Typography>{getLanguageString(organization.labels, 'nb')}</Typography>
<Typography>{organization.labels['en']}</Typography>
```

Syns ikke det er en deal breaker her da, så du kan vurdere selv om det som er dekker behovet :)